tests/lib/cli/parser_test.py

```diff
@@ -3,6 +3,7 @@
 import pytest
 
 from prowler.lib.cli.parser import ProwlerArgumentParser
+from prowler.lib.logger import logger
```

Comment:
The linter is raising that this value is unused, please remove it if so.

Reply:
Yes, forgot it there. I will remove it. Just need to fix a bunch of other tests

Reply:
If you need help with the tests tell me. Thanks!